chrome/injecttop.browserify.js

```diff
@@ -0,0 +1,49 @@
+window.browserpassDisplayTOTP = function(login) {
+  if(login.digits) {
+    var totpBox = document.createElement("div");
+
+    totpBox.style = `
```

Comment:
I found [chrome.tabs.insertCSS](https://developer.chrome.com/extensions/tabs#method-insertCSS), I think we should move all the styles in a some kind of `otp.css` and inject that file in the page.